Description: Remove need to run reconfigure target
Author: Samuel Henrique <samueloph@debian.org>
Forwarded: not-needed
Index: rsync/Makefile.in
===================================================================
--- rsync.orig/Makefile.in
+++ rsync/Makefile.in
@@ -193,15 +193,6 @@ configure.sh config.h.in: configure.ac a
 	else \
 	    echo "config.h.in has CHANGED."; \
 	fi
-	@if test -f configure.sh.old -o -f config.h.in.old; then \
-	    if test "$(MAKECMDGOALS)" = reconfigure; then \
-		echo 'Continuing with "make reconfigure".'; \
-	    else \
-		echo 'You may need to run:'; \
-		echo '  make reconfigure'; \
-		exit 1; \
-	    fi \
-	fi
 
 .PHONY: reconfigure
 reconfigure: configure.sh
@@ -215,17 +206,6 @@ restatus:
 Makefile: Makefile.in config.status configure.sh config.h.in
 	@if test -f Makefile; then cp -p Makefile Makefile.old; else touch Makefile.old; fi
 	@./config.status
-	@if diff Makefile Makefile.old >/dev/null 2>&1; then \
-	    echo "Makefile is unchanged."; \
-	    rm Makefile.old; \
-	else \
-	    if test "$(MAKECMDGOALS)" = reconfigure; then \
-		echo 'Continuing with "make reconfigure".'; \
-	    else \
-		echo "Makefile updated -- rerun your make command."; \
-		exit 1; \
-	    fi \
-	fi
 
 stunnel-rsyncd.conf: $(srcdir)/stunnel-rsyncd.conf.in Makefile
 	sed 's;\@bindir\@;$(bindir);g' <$(srcdir)/stunnel-rsyncd.conf.in >stunnel-rsyncd.conf
